Dockerfile

```diff
@@ -151,10 +146,17 @@ RUN echo 'deb http://ftp.fr.debian.org/debian/ bullseye main contrib non-free' >
     && ./rats \
     && cd .. \
     && rm -rf ./rats-2.4.tgz ./rats-2.4 \
+    && tar -zxvf cppcheck-1.90.tar.gz \
+    && cd cppcheck-1.90/ \
+    && make install MATCHCOMPILER="yes" FILESDIR="/usr/share/cppcheck" HAVE_RULES="yes" CXXFLAGS="-O2 -DNDEBUG -Wall -Wno-sign-compare -Wno-unused-function -Wno-deprecated-declarations" \
```

Comment:
Don't we need to do `configure` and/or `make` before `make install` ?

Reply:
No, we don't.
See the official [README](https://github.com/danmar/cppcheck/tree/1.90#gnu-make), autotools is not even supported but a wide range of build systems and build management systems are.

Reply:
Thanks for the answer, and the link to the documentation.
By the way, as a reviewer, I should be the one to resolve this conversation...